### PR TITLE
New package: brillo-1.2.1

### PR DESCRIPTION
**Pull request by CameronNemo**
https://gitlab.com/cameronnemo/brillo

### CODE REVIEW

File: srcpkgs/brillo/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'brillo'
+pkgname=brillo
+version=1.2.0
+revision=1
+build_style=gnu-makefile
+wrksrc="$pkgname-v$version"
+make_build_args="dist"
+make_install_args="install-dist"
+makedepends="help2man"
```
Comment (maxice8) [inline srcpkgs/brillo/template]: help2man is meant to be executed on the host system so move to `hostmakedepends`

**Comment by maxice8** [comment]
You can apply this against your current tree and it will fix the cross-compiling problem

```
diff --git a/srcpkgs/brillo/patches/0001-Makefile-use-HOSTPROG-to-generate-manpages-in-help2m.patch b/srcpkgs/brillo/patches/0001-Makefile-use-HOSTPROG-to-generate-manpages-in-help2m.patch
new file mode 100644
index 0000000000..497da9d361
--- /dev/null
+++ b/srcpkgs/brillo/patches/0001-Makefile-use-HOSTPROG-to-generate-manpages-in-help2m.patch
@@ -0,0 +1,31 @@
+From 1456791d05b554737f4137042011647ea163d7c8 Mon Sep 17 00:00:00 2001
+From: maxice8 <thinkabit.ukim@gmail.com>
+Date: Tue, 14 Aug 2018 04:37:06 -0300
+Subject: [PATCH] Makefile: use HOSTPROG to generate manpages in help2man
+
+---
+ Makefile | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index 7fec30b..cfb18fe 100644
+--- Makefile
++++ Makefile
+@@ -1,4 +1,5 @@
+ PROG := brillo
++HOSTPROG := $(PROG)
+ DESC := Backlight and Keyboard LED control tool
+ VENDOR := com.gitlab.CameronNemo
+ 
+@@ -28,7 +29,7 @@ man: brillo
+ ifndef HELP2MAN_VERSION
+ $(error "help2man is not installed")
+ endif
+-	help2man $(MANFLAGS) -o $(PROG).1 ./$(PROG)
++	help2man $(MANFLAGS) -o $(PROG).1 $(HOSTPROG)
+ 
+ polkit:
+ 	sed 's|@bindir@|$(PREFIX)/bin|g;s|@prog@|$(PROG)|g;s|@vendor@|$(VENDOR)|g;s|@desc@|$(DESC)|g' contrib/polkit.in >contrib/$(VENDOR).$(PROG).policy
+-- 
+2.18.0
+
diff --git a/srcpkgs/brillo/template b/srcpkgs/brillo/template
index ea775eb82d..b47856ea9e 100644
--- a/srcpkgs/brillo/template
+++ b/srcpkgs/brillo/template
@@ -14,3 +14,8 @@ license="GPL-3.0-or-later"
 homepage="https://gitlab.com/cameronnemo/brillo"
 distfiles="$homepage/-/archive/v$version/$wrksrc.tar.gz"
 checksum=e7783c35aa4fee063f5bae9c7aa4b41dfbaa2d5a2e3e6f98823783998f9481b2
+
+if [ "$CROSS_BUILD" ]; then
+	make_build_args+=" HOSTPROG=/usr/bin/brillo"
+	hostmakedepends+=" brillo"
+fi
```

**Comment by CameronNemo** [comment]
@maxice8 can you check this change I made to the makefile upstream? [link](https://gitlab.com/cameronnemo/brillo/commit/96e9adb1419937b3a7ee4806e7f0aa3867d14054)

**Comment by maxice8** [comment]
> @maxice8 can you check this change I made to the makefile upstream? link

the look unnecessarily complex. i'd rather just have `MANPROG` and it by default point to `./$(PROG)` would allow us to easily override it

File: srcpkgs/brillo/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'brillo'
+pkgname=brillo
+version="1.2.1"
```
Comment (maxice8) [inline srcpkgs/brillo/template]: don't quote version

**Comment by CameronNemo** [comment]
Thanks for your patience.

**Pull request merged**